test/index.test.js

```diff
@@ -0,0 +1,60 @@
+var should = require('should');
+var sinon = require('sinon');
+
+var ga = require('../src/index');
+
+var GLOBALS = ['ga', 'GoogleAnalyticsObject', 'window', 'document'];
+
+describe('react-ga', function() {
+  function getGaCalls() {
+    var q = (window.ga && window.ga.q) || [];
+
+    return q.map(function(args) {
+      return [].slice.call(args);
+    });
```

Comment:
We're actually taking advantage of the fact that Google Analytics' own [quick start code](https://developers.google.com/analytics/devguides/collection/analyticsjs/#quickstart) defines a `ga` function that just appends arguments to `ga.q` until the "real" GA script is loaded. This function essentially returns `ga.q` but makes sure all its arguments are actual arrays rather than [`arguments`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments) objects, making them easier to run our assertions on.